parm/config/config.resources

```diff
@@ -201,6 +201,12 @@ elif [ $step = "fcst" ]; then
         # Mediator only uses the atm model PETS
         export MEDPETS=$ATMPETS
 
+        if [[ $DO_AERO == "YES" ]]; then
+            # Aerosol model only uses the atm model PETS
+            export CHMPETS=$ATMPETS
```

Comment:
Is this ATMPETS or MEDPETS?  In the regression tests we've been setting up we've not used the write grid component ATM PETs so therefore have been using just the MEDPETS... What's the best thing here @rmontuoro ?

Reply:
This is before the write groups get added, so it is the same as MEDPETS